spec/notifications/new_aircraft_created_notification_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe(NewAircraftCreatedNotification, type: :model) do
+  # Must eagerly create a user first, or notifications will not be created
+  before { create(:user) }
```

Comment:
@srb- , Using  the create(:something) directly is kind of confusing, I think you can improve the  structure using `let` and `let!`: 
```ruby
  let!(:user) { create(:user) }
  let(:aircraft) { create(:aircraft) }
```
In this case ensures a user is created before each test, as the notifications depend on the existence of a user.
and the  `let(:aircraft) { create(:aircraft) }`  defines the aircraft creation, which will be executed when referenced. on each `it` block. so wen you need it a reference that is create with `let` not `!` you can call it in the it block sometine Rspec is lazy so you can do:
```ruby
RSpec.describe NewAircraftCreatedNotification, type: :model do
  let!(:user) { create(:user) }
  let(:aircraft) { create(:aircraft) }

  context "when aircraft are created" do
    it "creates database notifications" do
      expect { aircraft }.to change(Notification, :count).by(1)
    end

    it "creates email notifications" do
      expect { aircraft }.to change { UserMailer.deliveries.count }.by(1)
    end

    it "sends the proper email subject" do
      aircraft
      email = ActionMailer::Base.deliveries.last
      expected_subject = I18n.t("mailer.aircraft.created.subject", aircraft: aircraft)
      expect(email.subject).to eq(expected_subject)
    end
  end
end
```
Please test it, I didn't run it.

Reply:
Ah, I remember why I didn't do that. Rubocop: `'Do not use `let!` to setup objects not referenced in tests.'` Also, I find it odd to just reference the variable `aircraft` alone/no-op (last testcase) in a test like that? Is that common?

Reply:
Ohh, I remember now, you can do something like:
```ruby
  before { create(:user) }
  let(:aircraft) { create(:aircraft) }
```
before the `context  " " do`

Reply:
That works! Updated.